examples/index.html

```diff
@@ -61,29 +59,29 @@
           </div>
```

Comment:
It looks like the markup for the input group should change (no more `input-group-append`): https://getbootstrap.com/docs/5.2/forms/input-group/

There is no longer CSS for the `.input-group>.input-group-append>.input-group-text` border radius.

![image](https://user-images.githubusercontent.com/41094/184982731-c0457af0-82b7-4408-beba-9799517345c7.png)